### ISSUE TITLE
Cannot allow access for Spectacle to control computer

### ISSUE THREAD

**Issue by zmcguiregvl** [eczarny/spectacle#883]
I have a brand new mbp with Mojave installed and for some reason I cannot add Spectacle to the Accessibility > Privacy list. I have tried using the + button and nothing happens, I have tried dragging the application package (from the application dir) into the window and nothing happens. I have also went into the contents of the Spectacle package and tried to drag and drop the Contents > MacOS > Spectacle into the window and nothing happens. I have it unlocked so I should be able to add it but for some reason it doesn't want to get added. I'm now at a loss with what to try and I really like Spectacle and would normally use it on a daily basis. What else can I try to give Spectacle access? Thank you :)

**Comment by ldebruyn** [comment]
I have the same issue. It stopped me cold after upgrading to Mojave. Please advise a workaround, if it exists. 
Thanks!

**Comment by koome23** [comment]
I have the same issue as well. It worked on my older MacBookPro with Mojave after upgrading. However, I'm using the new MacBookPro with TouchBar which has this issue.

**Comment by bbedwell** [comment]
+1 also seeing this issue on mojave 10.14 with spectacle 1.2

**Comment by jlwestsr** [comment]
+2 Stopped working for me this morning after upgrading to Mojave 10.14

**Comment by bbedwell** [comment]
This is what I did to fix it: I made sure everything was updated (I am on 10.14.1) and then rebooted and spectacle was an option on the Accessibility list

**Comment by ldebruyn** [comment]
I have followed the same fix as @bbedwell and it worked, after two reboots.

**Comment by zmcguiregvl** [comment]
In my case it was TrendMicro that was running on my mac that was not allowing me to add any applications to 'Security & Privacy > Accessibility'

**Comment by koome23** [comment]
@zmcguiregvl I think you're right. I disabled TM Security on my Mac and that seems to show apps in the "Security & Privacay > Accessibility" panel.

**Comment by koome23** [comment]
Here's how I fixed it: 

1. Go to: https://esupport.trendmicro.com/en-us/home/pages/technical-support/antivirus-for-mac/1061005.aspx
2. Download the Diagnostic Toolkit: https://files.trendmicro.com/products/iTIS/9.0/Trend%20Micro%20Antivirus-9.0.1351.dmg
3. Run the Diagnostic Toolkit under **User Support** folder
4. In the General section, click the "Uninstall" button. This will temporary disable TM agent.
5. Reboot
6. Now you should see the Accessibility list of apps in the Privacy & Security. Spectacle is among the list and now you should be able to control the windows using your keyboard short cuts.

**Comment by MHolmes91** [comment]
I have this same issue. I am not using TrendMicro, but this is happening after re-imaging my machine using Carbon Copy Cloner. It appears as though my `/Library/Application Support/com.Apple.TCC/TCC.db` file doesn't exist.
